app/routes/_auth.dns-records.new.tsx

```diff
@@ -49,7 +49,7 @@ export default function NewDnsRecordRoute() {
   const actionData = useActionData();
 
   return (
-    <Container maxW="container.xl">
+    <Container maxW="container.xl" paddingY="6">
```

Comment:
Does this also add a madding to the top of the container? If so, is that intended?

Reply:
Good catch.  Probably want `pb={6}`

Reply:
Thanks for catching this.